src/manifest.json

```diff
@@ -40,6 +40,13 @@
 				"vendor/@cliqz/adblocker-webextension-cosmetics/adblocker.umd.min.js",
 				"content_scripts/cosmetics.js"
 			]
+		},
+		{
+			"matches": ["*://www.google.com/*", "*://google.com/*"],
+			"run_at": "document_start",
```

Comment:
maybe better to load at document_end or idle? so we can skip on the even listener?

Reply:
Works, changed.